person.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  def initialize(id, age, name = 'unknown', parent_permission = true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id, :parent_permission
+  attr_accessor :name, :age
+
+  def can_use_services?
+    if is_of_age? == true || @parent_permission == true
+      true
+    else
+      false
+    end
```

Comment:
- [ ] [OPTIONAL] If you want to make your project look a little cleaner you can just do like below:
```
def can_use_services?
    is_of_age?() || @parent_permission
end
```
if either one returns true the method will return true